code/_globalvars/traits/_traits.dm

```diff
@@ -532,6 +532,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_WIELDED" = TRAIT_WIELDED,
 		"TRAIT_BAKEABLE" = TRAIT_BAKEABLE,
 		"TRAIT_INSTANTLY_PROCESSES_BOULDERS" = TRAIT_INSTANTLY_PROCESSES_BOULDERS,
+		"TRAIT_CONTRABAND" = TRAIT_CONTRABAND,
```

Comment:
ABC

Reply:
Also wondering if we should just make this an object flag instead given how many things have this trait in `Initialize`, and that we don't really play with the trait sources of it much. Just a thought